File: 05. 문자열/10. 5622.py
import sys

def dial(n : str):
	if n == "A" or n == "B" or n == "C":
		return 3
	elif n == "D" or n == "E" or n == "F":
		return 4
	elif n == "G" or n == "H" or n == "I":
		return 5
	elif n == "J" or n == "K" or n == "L":
		return 6
	elif n == "M" or n == "N" or n == "O":
		return 7
	elif n == "P" or n == "Q" or n == "R" or n == "S":
		return 8
	elif n == "T" or n == "U" or n == "V":
		return 9
	elif n == "W" or n == "X" or n == "Y" or n == "Z":
		return 10
	else:
		return 11

s = sys.stdin.readline().rstrip()
result = 0
for i in range(0, len(s)):
	result += dial(s[i])
print(result)

File: C++/05. 문자열/10. 5622.cpp
#include <stdio.h>
#include <string.h>

int dial(char c) {
    if (c == 'A' || c == 'B' || c == 'C')
        return 3;
    else if (c == 'D' || c == 'E' || c == 'F')
        return 4;
    else if (c == 'G' || c == 'H' || c == 'I')
        return 5;
    else if (c == 'J' || c == 'K' || c == 'L')
        return 6;
    else if (c == 'M' || c == 'N' || c == 'O')
        return 7;
    else if (c == 'P' || c == 'Q' || c == 'R' || c == 'S')
        return 8;
    else if (c == 'T' || c == 'U' || c == 'V')
        return 9;
    else if (c == 'W' || c == 'X' || c == 'Y' || c == 'Z')
        return 10;
    else
        return 11;
}

int main() {
    int result = 0;
    char str[15];
    scanf("%s", str);

    for (int i = 0; i < strlen(str); i++) {
        result += dial(str[i]);
    }

    printf("%d", result);

    return 0;
}